packages/@aws-cdk/aws-codepipeline/test/test.cloudformation-pipeline-actions.ts

```diff
@@ -30,6 +30,7 @@ export = {
     stage: sourceStage,
     outputArtifactName: 'SourceArtifact',
     repository: repo,
+    pollForSourceChanges: true,
   });
 
   /** Build! */
```

Comment:
Missing unit test for the default and setting to `false`

Reply:
Sure.